dnSpy/dnSpy/MainApp/StartUpClass.cs

```diff
@@ -52,7 +52,13 @@ public static class StartUpClass {
 			if (!readSettings)
 				readSettings = AskReadSettings();
 
-			new App(readSettings, sw).Run();
+			var args = new AppCommandLineArgs();
+			if (args.DisplayHelp) {
```

Comment:
The curly braces can be removed, they're not used if it's just one statement inside the block.